src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -998,9 +1085,10 @@ class Class
             End Using
         End Function
 
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WpfTheory, CombinatorialData>
+        <Trait(Traits.Feature, Traits.Features.Completion)>
         <WorkItem(24432, "https://github.com/dotnet/roslyn/issues/24432")>
-        Public Async Function TestImplicitArrayInitialization2() As Task
+        Public Async Function TestImplicitArrayInitialization2(showCompletionInArgumentLists As Boolean) As Task
```

Comment:
love that we're testing both forms.  can you link to any changed behavior tests so taht i don't have to review every one of these? :)

Reply:
Posted at the bottom, I think really only SigHelp tests changed where now both sighelp and completion compete for keyboard focus.